manifests/manifest.yaml

```diff
@@ -148,7 +156,7 @@ webhooks:
+      caBundle: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURKRENDQWd5Z0F3SUJBZ0lSQU1IbWFOZTdrdjNDYXVyVjFCWG5ObkF3RFFZSktvWklodmNOQVFFTEJRQXcKSERFYU1CZ0dBMVVFQXhNUlkyOXphV2R1TFhkbFltaHZiMnN0WTJFd0hoY05NalF3TVRFeU1UUXlNREExV2hjTgpNelF3TVRBNU1UUXlNREExV2pBY01Sb3dHQVlEVlFRREV4RmpiM05wWjI0dGQyVmlhRzl2YXkxallUQ0NBU0l3CkRRWUpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFMbStSZXN5bWxVZlRpNW5lY0hRY2EwdTE2THIKTFExZkt3SUhJbVpscDB5OG83ZXd1UE55U1dwZ3lEOHFzVUlXeE1Wa3huQjM2ekdrTDloK0h2ekQ4RjF2ckRydApXcEF1ek5ieEwrc1pvd3ZZcjVRZWE3UXNSTDczVnovZncxWXVSeVk0Umg2elo4Rm1rTCswZjc1RWZpVzBLdjhiCnJwdHVucUtPdm9ERVVkL2F3R25ySXh4MC9KTWlIR091emVwWXlGWEtmMTRyOTlaMTQyMHRvdy9SUzZjT0t6dXEKSHlMTzJaeGJaQmtpeUY1TUVZbXk3L3JicU1kd29Ud29KOVJqeWVTQnlrbUwzbTQxQm9xeUIzckRBMkZObDlpdwpUc0JOTUJUOU1GYlhtWGp6aUZZeDZUU3lMRjQxODFQQmx2Sm5yeDBCMHlqbjduR0lmTGx4ZVdDV3NtTUNBd0VBCkFhTmhNRjh3RGdZRFZSMFBBUUgvQkFRREFnS2tNQjBHQTFVZEpRUVdNQlFHQ0NzR0FRVUZCd01CQmdnckJnRUYKQlFjREFqQVBCZ05WSFJNQkFmOEVCVEFEQVFIL01CMEdBMVVkRGdRV0JCU3NGUFU0OEhKNmdVVDF6aE9GQUt3awpXRGNVWVRBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQVFFQUxON2FFbUtMVGpRbncranU0VkViZjJ4OWFtNEdlSmhOCjhobHVxZFMvdENYNHJmU0w1R1pWNGxFZ0lpSXNBWm9hOW9TRmU4bERxZTNhYXZIMnBOUHBRK3lHUldpdjlKUUUKZnJ0MkpzNC8xUlUvWHpiL3pWNHBqT3VXSCtwY29acGFVY20rc1JYYjFQeG5sMjlZYVdkdDRRdHhFNHJKVDlVSQp2U3dHWG1FaTR6MXpTWlNGRFh0Y24vQkFpMzZ2UHpnWDZxcG0vajhSSEFIK21mWXhZcExPQ2RpdWVmTkZLajRlCmtzU01RZjI4NW4rRDNWVUhlRHNFb2JQWDk5enIrOW1PdStvN044OVVCUXU1VzJhNFJIMkxmaFRWSUYzS3h1T3gKRXB0c1JXY0Z6OHRMcnlHOTBNcXBkU3JmZ0ljZXdNalhyV25RMi9TOVkraXNMUENGU3FPRGpRPT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo="
```

Comment:
Same is valid for the caBundle - as it's patched via the generate script, we deploy it with an empty value and just patch it afterwards.